tests/Sniffs/TestCase.php

```diff
@@ -157,14 +157,7 @@ protected function getSniffClassName(): string
 
 	protected function getSniffPath(): string
```

Comment:
Can you please remove this method and create the reflection right in method `checkFile`? The same reflection is on line 33 so it would be nice to create it just once.